### PR TITLE
IAM-429-Automate approving and merging lib updates

### PR DESCRIPTION
**Pull request by bencekov**
This PR sets automerge on for automated-update-charm-libs PRs, and automatically approves them.

### CODE REVIEW

File: .github/workflows/renovate-approve.yaml [hunk 1]
```diff
@@ -5,12 +5,13 @@ on:
...
     branches:
       - "renovate/**"
+      - "automated-update-charm-libs"
 
 jobs:
   autoapprove:
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
+      - uses: actions/checkout@v3
```
Comment (nsklikas) [inline .github/workflows/renovate-approve.yaml:14]: Revert this change

File: .github/workflows/update_libs.yaml [hunk 2]
```diff
@@ -27,7 +27,7 @@ jobs:
     name: Update charm libraries
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
+      - uses: actions/checkout@v3
```
Comment (nsklikas) [inline .github/workflows/update_libs.yaml:30]: Revert this change

File: .github/workflows/renovate-approve.yaml (hunk 1 above)
Comment (natalian98) [reply .github/workflows/renovate-approve.yaml:14]: We moved to v4: 
```suggestion
      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
```

File: .github/workflows/update_libs.yaml (hunk 2 above)
Comment (bencekov) [reply .github/workflows/update_libs.yaml]: done

File: .github/workflows/renovate-approve.yaml (hunk 1 above)
Comment (bencekov) [reply .github/workflows/renovate-approve.yaml]: done

**Review by natalian98 — APPROVED** [review]
